README.md

```diff
@@ -214,10 +214,15 @@ The PIConGPU Team expresses its thanks to:
 - Stefan Tietze
 - Benjamin Worpitz
 - Erik Zenker
+- Alexander Matthes
 
 Kudos to everyone who helped!
 
 ********************************************************************************
 
 ![image of an lwfa](doc/images/lwfa_grey.png "LWFA")
```

Comment:
Well, I thought about replacing the old picture with this image:
![lwfa](https://raw.githubusercontent.com/ComputationalRadiationPhysics/isaac/master/example_renderings/picongpu_wakefield_5_6.png)